resources/openapi_v2.yaml

```
@@ -772,15 +935,15 @@ components:
           description: The person who sent the message
         time:
           type: string
-          description: The time which the message was sent
+          description: The message sent time
         hasRead:
           type: boolean
-          description: Whether the message is read or not
+          description: If the message has been read or not
         message:
           type: string
     Location:
       type: array
-      description: All the points in the polygon
+      description: This includes all the points in the polygon
```

Let's change this in other places as well.
```suggestion
      description: This includes all the points of the land in the form of a polygon.
```